server/server.js

```diff
@@ -4,39 +4,38 @@ const axios = require('axios').default;
 require("dotenv").config();
 const path = require('path');
 require('dotenv').config(); //??
-const db = require('../server/db/db-connection.js'); 
-const fetch = require("node-fetch"); 
+const db = require('../server/db/db-connection.js');
+const fetch = require("node-fetch");
 const { auth } = require('express-openid-connect');
 const apiKey = `${process.env.API_KEY}`;
 const REACT_BUILD_DIR = path.join(__dirname, '..', 'client', 'build');
 const app = express();
 
-
-
-
 const config = {
     authRequired: false,
     auth0Logout: true,
-    secret: process.env.SECRET,
-    baseURL: process.env.BASEURL,
+    baseURL: 'http://localhost:5001', // TODO: put this in your `.env` file
     clientID: process.env.CLIENTID,
-    issuerBaseURL: process.env.ISSUERBASEURL
+    issuerBaseURL: process.env.ISSUERBASEURL,
+    secret: process.env.SECRET,
 };
 
-
-const PORT = process.env.PORT || 3001;
+const PORT = process.env.PORT || 5001;
 app.use(cors());
 app.use(express.json());
+// app.use(express.static(REACT_BUILD_DIR));
 app.use(auth(config));
 
+
 //creates an endpoint for the route /api
 app.get('/', (req, res) => {
     // console.log(req.oidc.isAuthenticated());
-    res.sendFile(path.join(REACT_BUILD_DIR, 'index.html'));
+    // res.sendFile(path.join(REACT_BUILD_DIR, 'index.html'));
```

Comment:
I see why they made you create a `build` folder, because Auth0 will default go back to the homepage for the port specified, which in this case is 5001 since the `/login`, `/logout`, and `/callback` routes are all defined on the server. However, then it'll take you to `http://localhost:5001/`, which doesn't render anything so you had to render the `index.html` page for this endpoint. I think it's clearer just to redirect back to the client port (3000)